go/common/entity/entity.go

```diff
@@ -146,6 +147,11 @@ func (e *Entity) Save(baseDir string) error {
 	return os.WriteFile(entityPath, b, fileMode)
 }
 
+// StakingAddress returns the staking address for the entity.
+func (e *Entity) StakingAddress() staking.Address {
```

Comment:
The reason why this is not here is that we tried to avoid dependencies to non-common (e.g. staking) from common libraries.

Reply:
Ok, will remove both.